pkg/cli/public_key_server.go

```diff
@@ -119,7 +119,11 @@ func (c *PublicKeyServerCommand) RunUnstarted(ctx context.Context, args []string
 	keyServer := jvscrypto.NewKeyServer(ctx, kmsClient, c.cfg, h)
 
 	mux := http.NewServeMux()
-	mux.Handle("/healthz", healthcheck.HandleHTTPHealthCheck())
+	// This is the healthz checkpoint, but in cloud run url ending with z
+	// could be reserved. This is to fix that.
+	//
+	// See: https://cloud.google.com/run/docs/issues#ah
+	mux.Handle("/healthz-check", healthcheck.HandleHTTPHealthCheck())
```

Comment:
Why not `/health` or `/.operations/health`?

Reply:
Changed to `health`, previous I had some misunderstanding, I thought `health` is reserved for liveness probe.

Thank you!